index.html

```diff
@@ -7,10 +7,92 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
+    <link rel="stylesheet" type="text/css" href="style.css">
 </head>
 
 <body>
-    <!-- Add your markup here -->
+<!-- Main Content -->
+<main>
+<header>
+<div class="header">
+    <div class="container">
+        <img class="logo" src="/images/logo.png" alt="logo">
+    </div>
+    <div class="container-right">
+        <h1>Your Cake Mate</h1>
+        <h2>For Your Orders <br> 07403720162</h2>
+    </div>
+</header>
+<nav>  
+  <div class="topnav">
+    <a href="#">Home</a>
+    <a href="#">Cakes</a>
+    <a href="#">Ordering</a>
+    <a href="#">Lessons</a>
+    <a href="#">About</a>
+  </div>
+</nav> 
```

Comment:
As the navigation typically is always at the top of the page, this nav bar could live in the header element.